values.yaml

```diff
@@ -33,6 +38,12 @@ common:
     # "gp2" would be appropriate.
     class: "standard"
 
+# Create custom config file
+# You will need to tweak `command` & `args` to use it instead of default file
```

Comment:
I believe this is not an ideal solution. Normally helm charts are installed from helm repositories. Once we make that possible, this solution becomes a burden. Why exactly is this necessary?

Reply:
Why will it be a burden?
there is not difference between installgin from helm repo, or using files. Once the custom config is created you can edit it.
My main use case is to be able to use official ES docker images, but there is more use cases as it adds lot of flexibility.